daffodil-lib/src/test/resources/META-INF/services/org.apache.daffodil.api.ValidatorFactory

```diff
@@ -0,0 +1,17 @@
+#  Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Comment about the .keep file above that this interface won't let me put a comment on.

I think we don't need this .keep file.

Reply:
This dot keep is deleted in the other PR